src/govuk/components/pagination/template.njk

```diff
@@ -0,0 +1,71 @@
+{% set blockLevel = not params.items and (params.next or params.previous) %}
+
+<nav class="govuk-pagination {{- ' ' + params.classes if params.classes }} {{- ' govuk-pagination--block' if blockLevel }}" role="navigation" aria-label="{{ params.landmarkLabel | default("results") }} {%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}" {%- endfor -%}">
+  {%- if params.previous and params.previous.href -%}
+    <div class="govuk-pagination__prev">
+      <a class="govuk-link govuk-pagination__link" href="{{ params.previous.href }}" rel="prev" aria-label="{{ params.previous.visuallyHiddenText | default("Previous page") }}" {%- for attribute, value in params.previous.attributes %} {{ attribute }}="{{ value }}" {%- endfor -%}>
```

Comment:
I've just realised that because we're now using `aria-label` on the link, the `labelText` is no longer included in the accessible name for the link.

Over-simplifying somewhat, we currently end up with something like this:

```html
<a href="#" aria-label="Previous page">
    Previous<span class="govuk-visually-hidden">:</span> Title of previous page
</a>
```

So in the accessibility tree the label is just 'Previous page' 😕 

A couple of suggestions to get around this…

1. Don't use an `aria-label` on the previous/next links, as it's arguably less in need of the additional 'page' context than the page numbers are:
    ```html
    <a href="#">
        Previous<span class="govuk-visually-hidden">:</span> Title of previous page
    </a>
    ```

2. Change the default so that includes the `labelText` as well, which also means we can get rid of the visually hidden colon, but means that anyone customising `visuallyHiddenText` must also include the `labelText` themselves:

    ```html
    <a href="#" aria-label="Previous page: Title of previous page">
        Previous Title of previous page
    </a>
    ```